client/core/helpers.go

```diff
@@ -426,11 +427,13 @@ func (m *matchReader) StatusString() string {
 		return "(2 / 4) Second Swap Sent"
 	case order.MakerRedeemed:
 		if m.Side == order.Maker {
-			return "Match Complete"
+			return "Redemption Sent"
 		}
 		return "(3 / 4) Maker Redeemed"
 	case order.MatchComplete:
-		return "Match Complete"
+		return "Redemption Sent"
+	case db.MatchConfirmed:
```

Comment:
I'm not so sure about having `MatchConfirmed` in the db package or as an untyped int.  We're doing `switch m.Status` all with `case`s on an `order.MatchStatus`, and then one untyped int.
I think a `MatchStatus` field should always have a value from the `MatchStatus` enum.

The fact that that final status is not going to be used server-side does not particularly concern me.

Was there more discussion somewhere on the definition of this special status?

Reply:
It could still be an `order.MatchStatus` even in the **db** package. I prefer it to be defined in a client package, but I won't fight you for it.

Reply:
Meh.